scripts/src/components.ts

```diff
@@ -190,8 +192,7 @@ function normalizeEndpointForServer(endpoint: string): string {
 	while ((match = re.exec(endpoint)) !== null) {
 		endpoint = endpoint.replace(match[0], ":" + match[1]);
 	}
-
-	return endpoint;
+	return encodeUrl(endpoint);
```

Comment:
just wondering, what was the use case that triggered this change?

Reply:
hebrew commands...